### ISSUE TITLE
An inaccurate annotation

### ISSUE THREAD

**Issue by zpliu1126** [ablab/IsoQuant#200]
Hi~ Andrey,
I am using IsoQuant's v3.3.1 Reference-base model for Isoform annotation in polyploid species and updating my gff3 annotation file. To check the accuracy of the annotation, I visualized the alignment results of short and long reads in IGV. However, it seems that IsoQuant did not annotate the isoforms containing RI in AD1_HC04_A05_125890. But for the orthologous gene AD1_HC04_D05_496010 of AD1_HC04_A05_125890, IsoQuant was able to detect isoforms containing RI. I also checked the annotation 该方法 file before updating, and both genes only had isoforms annotated without RI; but IsoQuant only updated the corresponding annotations in AD1_HC04_D05_496010. I want to know what might be the reason for this situation?

### IGV show for AD1_HC04_A05_125890

![EK17L Z_R4OQF0)FO4ZW%XU](https://github.com/ablab/IsoQuant/assets/39325949/8bd4ebd0-9167-40d8-b5e3-c7a21b5f2238)

### IGV show for AD1_HC04_D05_496010

![P4X42PX$SV V3_$K3OHQOU5](https://github.com/ablab/IsoQuant/assets/39325949/a6dbf76a-09f1-4918-85d0-200b912d9c0e)

Best
zpliu

**Comment by andrewprzh** [comment]
@zpliu1126 

Indeed, an interesting case, but it's really hard to tell only using screenshots.
Could you maybe send me these transcripts, associated reads and their assignments from read_assignments.tsv?

Best
Andrey

**Comment by zpliu1126** [comment]
@andrewprzh 
"In the test document, HC04_A05031980 is the reference annotated gene that corresponds to the aforementioned AD1_HC04_A05_125890; while HC04_D05032280 corresponds to the aforementioned AD1_HC04_D05_496010.


### isoform annotated by IsoQuant
[test_models.txt](https://github.com/user-attachments/files/15855891/test_models.txt)


### read assignments
[Uploading test_read_assignments.txt…]()

**Comment by andrewprzh** [comment]
Thanks, will take a look!

**Comment by andrewprzh** [comment]
Sorry, it's been a while. Unfortunately, I am the only one working on IsoQuant now, and cannot handle all ongoing tasks. I hope I will get my hands on it.

Best
Andrey

**Comment by andrewprzh** [comment]
New IsoQuant 4.0 has a new algorithm for splice site correction in discovered transcript. I will close the issue for now, feel free to re-open if still relevant.
